test/parallel.jl

```diff
@@ -97,6 +97,11 @@ for p in procs(d)
     @test d[idxl] == p
 end
 
+d = SharedArray(Float64, (2,3))
+for x in 1:100000
+    d[:,2]
```

Comment:
That would of course be the other, better way of testing this, agreed. This way you just run out of file descriptors and crash, I guess...